velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -452,6 +455,14 @@ velox::fuzzer::ResultOrError AggregationFuzzerBase::execute(
           .config(core::QueryConfig::kMaxExtendedPartialAggregationMemory, "0");
     }
 
+    if (supportRowsStreaming) {
+      SCOPED_TESTVALUE_SET(
+          "facebook::velox::exec::Window::supportRowsStreaming",
```

Comment:
Can you add a testvalue in rows stream window build constructor for this? And expect the testvalue hook has been called or not?